db/migrate/20170729203107_add_views_count_to_questions.rb

```diff
@@ -0,0 +1,5 @@
+class AddViewsCountToQuestions < ActiveRecord::Migration
```

Comment:
Have you built out the views_count functionality yet? If not, I wouldn't mind taking a stab at it.